staging/src/k8s.io/apiserver/pkg/apis/flowcontrol/bootstrap/default.go

```diff
@@ -460,13 +482,14 @@ func kubeSystemServiceAccount(names ...string) []flowcontrol.Subject {
 	return subjects
 }
 
-func resourceRule(verbs []string, groups []string, resources []string, namespaces []string, clusterScoped bool) flowcontrol.ResourcePolicyRule {
+func resourceRule(verbs []string, groups []string, resources []string, namespaces []string, clusterScoped bool, resourceNames []string) flowcontrol.ResourcePolicyRule {
```

Comment:
It looks to me like we do not need to add a parameter here, every call site passes the same value through this parameter.